public/static/locales/en/faq.json

```diff
@@ -38,8 +38,8 @@
       "topic": "Understanding Circles - What’s Going on Here?",
       "question": "Why do Circles have value?",
       "answer": [
-        "Money is a sort of language. We agree on its meaning and that agreement is why it has value. It may not seem like it, but it works the same way with euros or with Bitcoin.",
-        "Circles’s value is based on the value that people put into it and offer in the community. If you sell your bike for circles, or translate for someone in the jobcentre, and they pay you with circles, this value comes in the system and stays in the community and circulates. We believe this is as real a ‘value’ behind our currency as the gold reserves behind euros, a value we have never seen and never will see."
+        "Money is a sort of language. We agree on its meaning and that agreement is why it has value. It may not seem like it, but it works the same way with Euros or with Bitcoin.",
```

Comment:
Think it makes sense to use `Euro` instead of `Euros` in all appearances, just as in Dollar/Bitcoin etc...